pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "py-xline"
+name = "pyxline"
```

Comment:
Why should we change the name?  It's better to keep aligned with the repo name.

Reply:
Done.